src/mm-input-mask/mm-input-mask.html

```diff
@@ -0,0 +1,30 @@
+<!--
+ * @license
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+-->
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../shared/js/datautils.html">
+<link rel="import" href="../shared/js/fontloader.html">
+<link rel="import" href="../shared/fonts/fonts.html"/>
+<link rel="import" href="../shared/js/measure.html">
+<link rel="import" href="../mm-input/mm-input.html">
+<link rel="import" href="../mm-item-notifier/mm-item-notifier.html">
+<polymer-element name="mm-input-mask" attributes="" extends="mm-input">
+	<template>
+		<link href="mm-input-mask.css" rel="stylesheet" type="text/css"/>
+		<div id="pseudo">
+			<template repeat="{{maskConfig}}">
+				<template if="{{type === 1}}">
```

Comment:
1 and 0 in this context seem a bit "magic".  I might recommend using strings for types/comparisons instead.